### ISSUE TITLE
Plugin request: SBS  Play

### ISSUE THREAD

**Issue by skilleek** [streamlink/streamlink#1106]
### Checklist

- [ ] This is a bug report.
- [ ] This is a feature request.
- [x] This is a plugin (improvement) request.
- [ ] I have read the contribution guidelines.

### Description

A plugin to catch streams from: http://play.sbs.co.kr/onair/pc/index.html

As always, many thanks for all your help!

**Comment by johnthecracker** [comment]
This is tall order.

All of their streams come from original source, this is like writting 50 plugins and putting them all into one.

It would be handy plugin.

**Comment by skilleek** [comment]
The problem of your url is that it expires quickly. At the moment, I would need only a plugin for SBS Power-FM-Audio.

From what I have seen, the token is generated using JWT (json web token). I could not figure out how to automaticaly renew the url stream after it expires.

Hope we will find a solution

**Comment by johnthecracker** [comment]
There is another problem.

With that url I get sound only.

**Comment by skilleek** [comment]
@johnthecracker  that's completely fine, since the online stream is only audio

**Comment by johnthecracker** [comment]
I find rtmp stream on your new site http://play.sbs.co.kr/onair/pc/index.html

Here is how you can play this with VLC. 

I get bad broadcast in Europe, maybe because resolution is 1920 x 1800.

`rtmpdump -v -r "rtmp://live.sbs.co.kr/sbsch6/_definst_/sbsch60.stream" ^
-a "sbsch6/_definst_/sbsch60.stream?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1MDA0NzUxNjMsInBhdGgiOiIvc2JzY2g2L19kZWZpbnN0Xy9zYnNjaDYwLnN0cmVhbSIsImR1cmF0aW9uIjoxODB9.obT2gHOmx8N5i2u-UVXuT7ZsGglBxnzsFnLrpMHhao0" ^
-f "WIN 26,0,0,137" ^
-W "http://play.sbs.co.kr/onair/pc/img/SBSOnAirPlayer.swf" ^
-p "http://play.sbs.co.kr/onair/pc/index.html" ^
-y "sbsch60.stream" | vlc.exe -
`

**Comment by Pzx5073** [comment]
Got some old data for SBS, from a old proyect I started a few months ago and I never finished

Example code:

load the main page.. and it calls http://api.sbs.co.kr/vod/_v1/Onair_Media_Auth_Security.jsp?playerType=mobile&channelPath=sbsch6pc&streamName=sbs1ch62.stream&rnd=199

That returns something like this:

x0pTLyTzYpVu+e1IY+PaGH6fIaATXvdYe9c53ahtrIWIbPxJooFoRzyFo/zH2H/+l0Jm0E4vAw73bsPZGMD7Uit0lH6UavU/06N6xdAal/gBM2hKcHO9cOF1O4s6iyLIEZ2LPjqUtREN2y9RTXFw8beEEESoyQo4/icrJ4J7Gg9ivJwWk9N+M0Yc2Dzz1eCChFiMLBvYAcpmucg2AUrEG/1ma9URfDjglwBu7W4wz5xkvSRFX8xLDhoEZC8gISM0rkFFFWJrGsLNIR/3ImInhrirf6eQx8CgBT6/CniV3Pn4DuZbOjDMv4lXaUvNMlxhFae/r8bLKJZZuri5YA41awaPonA+ljzcoCUekqzbKBL0Djg43WxVCnq/Mqm9HYs6mAwuPdHyEuk=

then i used: https://www.tools4noobs.com/online_tools/decrypt/

With Base64 enabled, select algorithm DES with mode ECB and And used the FIX key: 7d1ff4ea (I do not remember where I got it from) 

and then de decoder return something like this:

http://live.sbs.co.kr/sbsch6pc/_definst_/sbsch62.stream/playlist.m3u8?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOiIxNDkzNzY4MjIzIiwiZHVyYXRpb24iOiItMSIsInBhdGgiOiIvc2JzY2g2cGMvX2RlZmluc3RfL3Nic2NoNjIuc3RyZWFtIiwiRVhQIjoiMjAxNzA3MjUxNDM3MDMifQ.l8jpVtFKGYSzywgQy1RVAer-VubJY5VN4Azi_Cgm4Mc

Which would be the URL with the included token that lasts a few seconds to use.

I hope someone can use the information to build a Plugin.

Greetings.
